src/main/java/org/embulk/input/jira/util/JiraUtil.java

```diff
@@ -96,26 +97,25 @@ public static void validateTaskConfig(final PluginTask task)
      * For getting the timestamp value of the node
      * Sometime if the parser could not parse the value then return null
      * */
-    private static Timestamp getTimestampValue(PluginTask task, Column column, String value)
+    private static Timestamp getTimestampValue(final PluginTask task, final Column column, final String value)
     {
-        List<ColumnConfig> columnConfigs = task.getColumns().getColumns();
+        final List<ColumnConfig> columnConfigs = task.getColumns().getColumns();
         String pattern = DEFAULT_TIMESTAMP_PATTERN;
-        for (ColumnConfig config : columnConfigs) {
+        for (final ColumnConfig config : columnConfigs) {
+            final ConfigSource columnConfig = config.getConfigSource();
             if (config.getName().equals(column.getName())
-                    && config.getConfigSource() != null
-                    && config.getConfigSource().getObjectNode() != null
-                    && config.getConfigSource().getObjectNode().get("format") != null
-                    && config.getConfigSource().getObjectNode().get("format").isTextual()) {
-                pattern = config.getConfigSource().getObjectNode().get("format").asText();
+                    && columnConfig != null
+                    && columnConfig.has("format")) {
+                pattern = columnConfig.get(String.class, "format");
```

Comment:
is `format` here always text that leads to the condition of `isTextual` becomes useless?

Reply:
Typically in happy case, it would be text but sometime user could do an alternative approach (i.e try to input it as `json`), so in the past I use this approach to prevent the incidents. 
But right now the logic is removed.

Reply:
fine for me.